apps/web/src/components/Player/PlayerMenuBar/PlayerMenuBar.tsx

```diff
@@ -71,6 +82,14 @@ export default function PlayerMenuBar({ roomName, disconnect }: { roomName?: str
           </Grid>
         </Grid>
 
+        <Grid item>
+          <Grid container justifyContent="center">
+            <Button onClick={() => toggleParticipantWindow()} startIcon={<ParticipantIcon />}>
+              Participants
+            </Button>
+          </Grid>
+        </Grid>
```

Comment:
I'm not sure that this button needs to be in its own grid. I think we can put it in the same grid as the Hand Raise button.
```
        <Grid item>
          <Grid container justifyContent="center">
            <Button onClick={handleRaiseHand} startIcon={isHandRaised ? <LowerHandIcon /> : <RaiseHandIcon />}>
              {isHandRaised ? 'Lower Hand' : 'Raise Hand'}
            </Button>
            <Button onClick={() => toggleParticipantWindow()} startIcon={<ParticipantIcon />}>
              Participants
            </Button>
          </Grid>
        </Grid>
```